datadog_checks_downloader/datadog_checks/downloader/cli.py

```diff
@@ -113,7 +116,10 @@ def download():
                 sys.exit(1)
 
     tuf_downloader = TUFDownloader(
-        repository_url_prefix=repository_url_prefix, root_layout_type=root_layout_type, verbose=verbose
+        repository_url_prefix=repository_url_prefix,
+        root_layout_type=root_layout_type,
+        ignore_python_version=ignore_python_version,
```

Comment:
Do we need to keep it in the object state? Why not just pass it to the one function that uses it in line 124 below? 🙂

Reply:
Might be useful in future

Reply:
addressed